nilearn/datasets/atlas.py

```diff
@@ -773,3 +772,117 @@ def fetch_coords_dosenbach_2010():
                   networks=out_csv['network'], description=fdescr)
 
     return Bunch(**params)
+
+
+def fetch_atlas_juelich_historical(atlas_name, data_dir=None,
+                                   symmetric_split=False,
+                                   resume=True, verbose=1):
+    """Loads Juelich historical atlases from FSL if installed.
+
+    Parameters
+    ----------
+    atlas_name: string
+        Name of atlas to load. Can be:
+        cort-maxprob-thr0-1mm,  cort-maxprob-thr0-2mm,
+        cort-maxprob-thr25-1mm, cort-maxprob-thr25-2mm,
+        cort-maxprob-thr50-1mm, cort-maxprob-thr50-2mm,
+        sub-maxprob-thr0-1mm,  sub-maxprob-thr0-2mm,
+        sub-maxprob-thr25-1mm, sub-maxprob-thr25-2mm,
+        sub-maxprob-thr50-1mm, sub-maxprob-thr50-2mm,
+        cort-prob-1mm, cort-prob-2mm,
+        sub-prob-1mm, sub-prob-2mm
+
+    data_dir: string, optional
+        Path of data directory. It can be FSL installation directory
+        (which is dependent on your installation).
+
+    symmetric_split: bool, optional
+        If True, split every symmetric region in left and right parts.
+        Effectively doubles the number of regions. Default: False.
+        Not implemented for probabilistic atlas (*-prob-* atlases)
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, keys are:
+
+        - "maps": nibabel.Nifti1Image, 4D maps if a probabilistic atlas is
+          requested and 3D labels if a maximum probabilistic atlas was
+          requested.
+
+        - "labels": string list, labels of the regions in the atlas.
+    """
+    atlas_items = ('maxprob-thr0-1mm', 'maxprob-thr0-2mm', 'prob-1mm',
+                   'prob-2mm', 'maxprob-thr25-1mm', 'maxprob-thr25-2mm',
+                   'maxprob-thr50-1mm', 'maxprob-thr50-2mm')
+
+    if atlas_name not in atlas_items:
+        raise ValueError("Invalid atlas name: {0}. Please chose an atlas "
+                         "among:\n{1}".format(atlas_name,
+                                              '\n'.join(atlas_items)))
+
+    # For practical reasons, we mimic the FSL data directory here.
+    dataset_name = 'fsl'
+    # Environment variables
+    default_paths = []
+    for env_var in ['FSL_DIR', 'FSLDIR']:
+        path = os.getenv(env_var)
+        if path is not None:
+            default_paths.extend(path.split(':'))
+    if len(default_paths) == 0:
+        raise ValueError("Please export FSL_DIR or FSLDIR in your environ."
+                         " This can be done by sourcing the fsl.sh "
+                         "configuration file, for example.")
```

Comment:
Ya it's related. Saw that too. That's the SPM pluggin for exploring the
atlases. What we really need are the atlases themselves.

On Thu, Jun 23, 2016 at 12:29 PM, bthirion <notifications@github.com> wrote:

> In nilearn/datasets/atlas.py
> <https://github.com/nilearn/nilearn/pull/1146#discussion_r68211338>:
>
> > +        raise ValueError("Invalid atlas name: {0}. Please chose an atlas "
> > +                         "among:\n{1}".format(atlas_name,
> > +                                              '\n'.join(atlas_items)))
> > +
> > +    # For practical reasons, we mimic the FSL data directory here.
> > +    dataset_name = 'fsl'
> > +    # Environment variables
> > +    default_paths = []
> > +    for env_var in ['FSL_DIR', 'FSLDIR']:
> > +        path = os.getenv(env_var)
> > +        if path is not None:
> > +            default_paths.extend(path.split(':'))
> > +    if len(default_paths) == 0:
> > +        raise ValueError("Please export FSL_DIR or FSLDIR in your environ."
> > +                         " This can be done by sourcing the fsl.sh "
> > +                         "configuration file, for example.")
>
> I don't know whether this is related to the SPM toolbox released by
> Julich. Maybe there is also some information on licensing there.
> http://www.fz-juelich.de/SharedDocs/Downloads/INM/INM-1/DE/Toolbox/Toolbox_22c.html;jsessionid=5FA071E64571794B894853F6D457F67A?nn=1090980
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nilearn/nilearn/pull/1146/files/527f4be282bb205d8b36a5befde1974c86b70c4a#r68211338>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAms1Gwp9KSuujFuIELJSqL7T7gR_gw4ks5qOl_6gaJpZM4I8mip>
> .
>



-- 
Elvis Dohmatob,
*sent from my  potato...*
http://dohmatob.blogspot.fr/, https://team.inria.fr/parietal/elvis/,
https://github.com/dohmatob